### ISSUE TITLE
[Package Request]: MSI Center

### ISSUE THREAD

**Issue by Cyberavater** [microsoft/winget-pkgs#195084]
### How can we help?

None

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.msi.com/Landing/MSI-Center
Publisher: MSI
Package Name: MSI-Center
Description: Management app for MSI motherboard.
Package Version: N/A
Installer Url: https://download.msi.com/uti_exe/desktop/MSI-Center.zip

**Comment by Dragon1573** [comment]
It seems the package is a "quadra-layer nested" installer file as follows:

```text
MSI-Center.zip (Downloaded ZIP archive)
└── MSI Center_2.0.45.0.exe (Install wizard executable inside the archive)
    └── MSI Center (Shortcut in Startup Panel after installation)
        └─ (An extra step for clicking the icon and waiting MSI Center to be installed and configured properly)
```

Also, the nested EXE install wizard NOES NOT write any ARP entries to _Windows Registry_. This might cause "infinite upgrade loop" in the future.

**Comment by Cyberavater** [comment]
Does it mean it cant be added to Winget?

**Comment by Dragon1573** [comment]
> Does it mean it cant be added to Winget?

Yes ✅. I think `winget.exe` does not support such kind of packages.
